src/handlers/gestures/GestureDetector/index.tsx

```diff
@@ -164,10 +164,10 @@ export const GestureDetector = (props: GestureDetectorProps) => {
   }, []);
 
   useEffect(() => {
-    if (!state.firstRender) {
-      updateAttachedGestures();
-    } else {
+    if (state.firstRender) {
       state.firstRender = false;
+    } else {
+      updateAttachedGestures();
```

Comment:
I'm a bit confused, isn't it the same as before? 😅

Reply:
Yes, but the logic isn't inverted, so it should be (at least is for me 😅) easier to understand at a glance.